tests/e2e/configuration/configuration_check.py

```diff
@@ -25,8 +24,23 @@ def test_does_default_config_match():
     cursor.execute("SHOW CONFIG")
     config = cursor.fetchall()
 
-    for flag in config:
-        assert config[1] == default_config.startup_config[1]
+    assert len(config) == len(default_config.startup_config)
+
+    for idx in range(len(config)):
+        current_flag = ()
+
+        for flag in config:
+            if flag[0] == default_config.startup_config[idx][0]:
+                # This will always depend on the number of processing units available on the given machine
+                if flag[0] == "bolt_num_workers":
+                    continue
+
+                current_flag = flag
+                break
+
+        if current_flag != ():
+            assert len(current_flag) == len(default_config.startup_config[idx])
+            assert current_flag[1] == default_config.startup_config[idx][1]
```

Comment:
In default config you have listed the current values and descriptions. I either simplify that structure or add checks here.

Reply:
I am not sure if I'm understand the reasoning behind this. The purpose of this test is to check if the default config have changed or not, as it was requested by @kostasrim.
"either simplify that structure or add checks here" - Do you mean to simplify the list of tuples that is in `/configuration/default_config.py` maybe into just a list of values? Should it just be a list of `default_values`?

I could check against the description, if that changed or not,
I see no point in checking against the name of the flags because I need them to make sure I find the individual tuples in case the different machines shuffle the order of them(happens),
So basically I should only remove the `current_value`s from the `default_config`.

To sum it up, on individual flags
1. Check if the `default_config` has changed or not
2. Check if the `description` has changed or not
3. Remove the `current_config` from the `default_config`

is that correct? @antaljanosbenjamin

Reply:
The test is not about checking if the default config has changed or not but rather if the implementation of `SHOW CONFIG` does what is intended to do. You start memgraph with a certain configuration and the values of that configuration are displayed in the `current_config` column of the `SHOW CONFIG` + the default values and the description. I would make `startup_config` a dictionary (description will be the key), and then in the test I would look the `default` && `current` values of each description. That way you can assert/cross-reference the expected configuration and the one you fetched from `SHOW CONFIG`

Reply:
**Conlusion**:
I will create a dictionary out of the `startup_config`, use the name of the flag as the key, and a tuple of `default_value`, `current_value`, `description` as the key.